### PR TITLE
1682 cookie notice

### LINKED ISSUE #1682: Implement the cookie notice
Now that you can get to xPub directly we need to implement the cookie notice.

The cookie covers all subdomains of `elifesciences.org`  and the hiding persists across these because of how the cookie is set.

Implementation can be found here:
https://github.com/elifesciences/pattern-library/blob/develop/assets/js/components/CookieOverlay.js

/cc @hdrury1 @davidcmoulton

### PR DESCRIPTION
**Pull request by Cooryd**
#### Background

Adds a cookie notice to the footer of the screen that won't show once accepted.

#### Any relevant tickets

closes #1682

#### How has this been tested?
**Reviewers** ensure that these test requirements are also reviewed.

Please indicate the need for this change to be tested. If not please justify, if so then at what levels. 

- [x] Unit / Integration tests
- [x] Browser tests

#### UI Changes

- [ ] Has been reviewed against Designs
- [x] Necessary updates to styleguide

### CODE REVIEW

File: app/components/global/CookieNotice/index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { Box } from '@rebass/grid'
+import { Button } from '@pubsweet/ui'
+import FooterMessage from 'ui/atoms/FooterMessage'
+import NativeLink from 'ui/atoms/NativeLink'
+import Helpers from './helpers'
+
```
Comment (diversemix) [inline app/components/global/CookieNotice/index.js:7]: Just a suggestion ... do we want to reference that this cookie is the one from Journal? @Cooryd

**Review by diversemix — APPROVED** [review]

File: app/components/global/CookieNotice/index.js (hunk 1 above)
Comment (Cooryd) [reply app/components/global/CookieNotice/index.js:7]: I'm not sure I can use the same cookie across both sites as they're on different domains. This one is specific to reviewer.elifesciences.org

**Comment by hdrury1** [comment]
FYI, see below for our implementation:

Screen Shot 2019-03-19 at 13 17 18

vs Journal implementation:

Screen Shot 2019-03-19 at 13 17 25

File: app/components/global/CookieNotice/index.js (hunk 1 above)
Comment (diversemix) [reply app/components/global/CookieNotice/index.js:7]: Ah what I meant was via some sort of comment in way of explanation to a dev.

Comment (Cooryd) [reply app/components/global/CookieNotice/index.js:7]: I'll leave a comment that the design and wording are copied from journal

**Comment by hdrury1** [comment]
Is the sizing difference due to a minimum button width we have set that Journal doesn't?

**Review by hdrury1 — APPROVED** [review]

**Review by hdrury1 — APPROVED** [review]

**Pull request merged**